src/Sentry/SentryOptions.cs

```diff
@@ -59,12 +59,18 @@ public bool IsGlobalModeEnabled
         }
     }
 #else
+    private bool? _isGlobalModeEnabled;
+
     /// <summary>
     /// Specifies whether to use global scope management mode.
     /// Should be <c>true</c> for client applications and <c>false</c> for server applications.
     /// The default is <c>false</c>. The default for Blazor WASM, MAUI, and Mobile apps is <c>true</c>.
     /// </summary>
-    public bool IsGlobalModeEnabled { get; set; } = Runtime.Current.IsBrowserWasm();
+    public bool IsGlobalModeEnabled
+    {
+        get => _isGlobalModeEnabled ?? Runtime.Current.IsBrowserWasm();
```

Comment:
I think this is called often so we could cache the value here on first access:

```suggestion
        get => _isGlobalModeEnabled ??= Runtime.Current.IsBrowserWasm();
```